roboteam_ai/src/skills/InterceptBall.cpp

```diff
@@ -163,11 +170,12 @@ Vector2 InterceptBall::computeInterceptPoint(Vector2 startBall, Vector2 endBall)
         else {
             // if the Line does not intercept it usually means the ball is coming from one of the corners-ish to the keeper
             // For now we pick the closest point to the (predicted) line of the ball
-            interceptionPoint = Vector2(robot->pos).project(startBall, endBall); }
+            interceptionPoint = Vector2(robot->pos).project(startBall, endBall);
```

Comment:
you should probably make startball, endball a line and project the robot->pos onto that